app/about.html

```diff
@@ -44,52 +44,104 @@
 
   <main>
     <!-- About section -->
-    <section id="about">
+    <section id="about" class="inset-1220">
       <h1 class="uppercase section-title">About us</h1>
 
-      <div class="team-member">
-        <img src="./images/desktop.gif" alt="Sia Karamalegos">
-        <h2>Sia Karamalegos</h2>
-        <h3>Founder &amp; Lead Developer</h3>
-        <p>bio...</p>
-        <ul class="social">
-          <li>
-            <a href="#">github</a>
-          </li>
-          <li>
-            <a href="#">linkedin</a>
-          </li>
-          <li>
-            <a href="#">medium</a>
-          </li>
-          <li>
-            <a href="#">twitter</a>
-          </li>
-        </ul>
+      <div class="bio">
+        <div class="bio-image">
+          <img src="images/sia.png" alt="Sia head shot">
+        </div>
+        <div class="bio-description">
+          <h2>Sia Karamalegos</h2>
+          <h3>Founder &amp; Lead Developer</h3>
+          <p>Sia has over 15 years of experience in technology, strategy, and operations across multiple industries. She loves
+            building applications that help make people's lives easier. She also loves making developer's lives easier through
```

Comment:
Grammar thing: `developer's` -> `developers'` so it's the plural form.

Reply:
good catch - fixed